sage-ios/SAGE/SAGE/RequestHoursView.swift

```diff
@@ -71,12 +75,14 @@ class RequestHoursView: UIView {
         self.dateFormatter.dateStyle = .NoStyle
         self.dateFormatter.timeStyle = .ShortStyle
```

Comment:
wait why are you setting these properties here instead of in viewdidload or some other method?